client/galaxy/scripts/components/Workflow/Editor/Node.vue

```diff
@@ -69,9 +91,20 @@ export default {
             }
             return null;
         },
+        popoverId() {
+            return `popover-${this.id}-${this._uid}`;
+        },
         canClone() {
             return this.type != "subworkflow";
         },
+        isEnabled() {
+            const Galaxy = getGalaxyInstance();
+            const isRecommendationEnabled = Galaxy.config.enable_tool_recommendations;
+            if (isRecommendationEnabled === true) {
```

Comment:
Test should be just `if (isRecommendationEnabled)` right? Or is there a special case where this explicit test is required?

Reply:
Fixed it, thanks!